src/main/java/mate/academy/model/Order.java

```diff
@@ -1,8 +1,11 @@
 package mate.academy.model;
 
 import java.util.List;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Order {
+    private static final Logger logger = LogManager.getLogger(Order.class);
```

Comment:
Same for all models, please remove, logging required during some business logic flow and model should not help service to perform logging.
Let's imagine we have 100 classes that uses user and we wish to add logging for all of them. Our User class will have 100 different methods for proper logging for each situation?